examples/ota-requestor-app/openiotsdk/main/main_ns.cpp

```diff
@@ -66,22 +66,27 @@ int main()
         return EXIT_FAILURE;
     }
 
-    static const osThreadAttr_t thread_attr = {
-        .stack_size = 16 * 1024 // Allocate enough stack for app thread
-    };
-
-    osThreadId_t appThread = osThreadNew(app_thread, NULL, &thread_attr);
-    if (appThread == NULL)
+    if (openiotsdk_network_init(true))
```

Comment:
I guess the app_thread should be removed in this case too.

Reply:
done